.gitignore

```diff
@@ -1,3 +0,0 @@
-Centos-68-64bit.vdi
-isos/*
-vdis/*
```

Comment:
.gitignore lives in the root directory, why has this been moved?
FYI: .gitignore will ignore these files, so they won't appear as "Untracked" files when you do a `git status`, so you don't add them by mistake.

Reply:
"Centos-68-64bit.vdi" is not generated in source files anymore.
"isos/" and "vdis/" are directories used by the scripts in "runners-template-vms-creating/" directory. It's intended for them to always be empty (from the repository perspective), because they should be populated manually by the project user (it's described in "runners-template-vms-creating/README"). 
However, they should be present in the repository, because they are referred to from scripts and README. 
To add a directory to GIT, there must be at least 1 file in it. So one option to achieve the effect of having an "empty directory" is to put ".gitignore" in it, that ignores all files apart from itself.
That's why "isos/.gitignore" and "vdis/.gitignore".

Reply:
ok, sounds good to me